src/ape/contracts/base.py

```diff
@@ -88,17 +88,20 @@ def __call__(self, *args, **kwargs) -> Any:
         txn.chain_id = self.provider.network.chain_id
 
         raw_output = self.provider.send_call(txn)
-        tuple_output = self.provider.network.ecosystem.decode_calldata(
+        output = self.provider.network.ecosystem.decode_return_data(
             self.abi,
             raw_output,
         )
 
+        # TODO: Handle struct output
+        if not isinstance(output, (list, tuple)):
+            return output
+
         # NOTE: Returns a tuple, so make sure to handle all the cases
-        if len(tuple_output) < 2:
-            return tuple_output[0] if len(tuple_output) == 1 else None
+        elif len(output) < 2:
+            return output[0] if len(output) == 1 else None
```

Comment:
This might not be the correct decision to make, depending on if the output is a single-member struct type, similar to resolving #574

Reply:
I did leave the TODO in there for this... You think that issue needs to be fixed in this PR?